improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -313,13 +314,18 @@ def _calculate_wet_bulb_temperature(
 
         # Initialise wet bulb temperature increment
         delta_wbt = 10. * np.broadcast_to(self.precision, temperature.shape)
-        delta_wbt_prev = None
 
         # Iterate to find the wet bulb temperature, using temperature as first
         # guess
         wbt_data = temperature.copy()
         iteration = 0
-        while (np.abs(delta_wbt) > self.precision).any():
+        while (np.abs(delta_wbt) > self.precision).any() \
+                and iteration < self.maximum_iterations:
+
+            if iteration > 0:
+                # Update saturation mixing ratio and iterators
```

Comment:
Don't need this comment any more (and it's become detached from what it's referring to!).

Reply:
Comment deleted.